src/jti_background.js

```diff
@@ -445,19 +446,31 @@ function JSONtoTemplateData (templates) {
     return formattedTemplates;
 }
 
-function JSONtoDomainData (domains) {
-    var nextID = getNextID(domains);
+function JSONtoDomainData (domains, callback) {
+    var nextID = 1;
```

Comment:
Why hardcode this to one? What was wrong with getNextID?

Reply:
When JSONtoDomainData is called, we replace all the domain data. Therefore we start with a new set of IDs.

e.g. if I have 8 existing domains, then import from a file, in the old version the IDs would start at 9. Now they'll start at 1.

Reply:
nevermind, what I said above is nonsense. I'll dig into it.

Reply:
getNextID and =1 do the same thing here. getNextID() is heavier (needs a function call and some internal logic), but it's probably clearer what the intent of the program is, so I'll use it.